charts/rawls/templates/migration/role.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.migration.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: rawls-migration-role
+  labels:
+    {{- include "rawls.labels" . | nindent 4 }}
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocr.argoproj.io/hook-delete-policy: BeforeHookCreation
+    argocd.argoproj.io/sync-wave: "-1"
```

Comment:
Is this to ensure the rbac resources get created before the acutal job?

I actually like creating and destroying these on each run of the hook. Helps reduce clutter in the argo ui.

Reply:
> Is this to ensure the rbac resources get created before the acutal job?
Yeah. Actually testing right now to see if the "HookSucceeded" deletion policy would work for us, leaving the resources around until the job was done.

Reply:
Believe it or not, it looks like the "HookSucceeded" policy secretly isn't supported on non-job resources. Literally a no-op. So unfortunately it looks like we might live with the clutter.

Reply:
In that case I'm pro Chelsea's suggestion of just reusing the existing rawls RBAC resources.

Reply:
I don't know if we want to sacrifice health checks and stuff on the actual Rawls RBAC stuff: https://github.com/broadinstitute/terra-helm/pull/387#discussion_r651290417